### ISSUE TITLE
Update to latest detection code in karma-detect-browsers

### ISSUE THREAD

**Issue by jvilk** [Subash/detect-browsers#1]
I made some updates to the browser detection code in `karma-detect-browsers`. There are also some other updates there you should pull in, e.g. Firefox Aurora's executable on Linux is `FirefoxAurora`, not `Firefox Aurora`.

Also, I'm talking with the developers about the possibility of making this an official dependency of `karma-detect-browsers` [in this issue](https://github.com/litixsoft/karma-detect-browsers/issues/10); I'm waiting for clarification from the developers. If this is a responsibility that you do not want, I am willing to take over stewardship of the module.

Let me know!

**Comment by Subash** [comment]
Sure, If you want to maintain this module. I can transfer the ownership.

**Comment by jvilk** [comment]
Yup, I'd be happy to. Should I take the repository, or start fresh? [I'm jvilk on NPM.](https://www.npmjs.com/~jvilk)
